docs/configuration/radio/device.mdx

```diff
@@ -11,24 +11,34 @@ The device config options are: Role, Serial Output, and Debug Log. Device config
-| `CLIENT_HIDDEN`  |                                                                                                                            Client Hidden - Used for nodes that "only speak when spoken to." Turns off all of the routine broadcasts but allows for ad-hoc communication. Still rebroadcasts, but with local only rebroadcast mode (known meshes only). Can be used for clandestine operation or to dramatically reduce airtime / power consumption                                                                                                                             |
-| `LOST_AND_FOUND` |                                                                                                                                                                                                  Lost and Found - Used to automatically send a text message with current position at frequent intervals to the primary channel for the device: "I'm lost! Position: lat / long"                                                                                                                                                                                                   |
+### Roles
+
+Device Role   |Description                                                                               |Best Uses                                                                                                                             
+--------------|------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------
+CLIENT        |Standard device for communication over the network, supports client apps.                 |General use for individuals needing to communicate over the Meshtastic network with support for client applications.                  
+CLIENT_MUTE   |Similar to CLIENT but does not forward packets.                                           |Situations where a node needs to participate in the network without assisting in packet routing, reducing network load.               
+ROUTER        |Devices placed for optimal routing, does not interact with client applications.           |Strategic placement in high-coverage areas to enhance network routing capabilities without the need for client interaction.           
+ROUTER_CLIENT |Acts as both a router and client device.                                                  |Devices that need to serve as both a communication point for users and a routing node to enhance network coverage.                    
+REPEATER      |Focuses on extending range with minimal network overhead.                                 |Extending the network's range by relaying messages without adding extra network overhead. Ideal for cleaner network topology.         
```

Comment:
@charminULTRA any thoughts on these?

`Focuses on extending range with minimal network overhead.` -> `Dedicated node that focuses solely on extending mesh network coverage by relaying messages, without appearing in the network's active node list or being directly addressable.`

`Extending the network's range by relaying messages without adding extra network overhead. Ideal for cleaner network topology.` -> `Best positioned in elevated or strategic locations to maximize the mesh network's overall coverage and connectivity.`

Reply:
@tekstrand I like those edits! Agreed.

Overall I like your versions better, as I'm generally on the side of making our Docs almost entirely in Plain English, but others may chime in that they want to keep some of the technical information, esp for Tracker and Sensor.

One small edit to Sensor I'll put in the Google Doc.

Reply:
I am also expecting people to request keeping the technical documentation. If that happens, my suggestion is going to be putting the information in it's own dedicated table.

Reply:
> I am also expecting people to request keeping the technical documentation. If that happens, my suggestion is going to be putting the information in it's own dedicated table.

I think once the "plain english" gets tightened up, we could ask some devs to chime in on the technical side.  I think some tech info would be beneficial otherwise it's "take our word for it, you want this role"  For example "When used in conjunction with power.is_power_saving = true, nodes will wake up, send position, and then sleep for position.position_broadcast_secs seconds."  This needs to be mentioned somewhere or this feature wont be used.